cmake/developer_package.cmake

```diff
@@ -214,6 +216,22 @@ set_property(GLOBAL PROPERTY USE_FOLDERS ON)
 
 set(CMAKE_POLICY_DEFAULT_CMP0054 NEW)
 
+# LTO
+
+set(CMAKE_POLICY_DEFAULT_CMP0069 NEW)
```

Comment:
`cmake_minimum_required()` setups policy settings according to specified version. No need to change them again.

Check examples from this manual page:
```
cmake --help-policy CMP0069
```

Reply:
but looks like `cmake_minimum_required` does not propagate values outside the `find_package` (this file is used in InferenceEngineDeveloperpackageConfig.cmake)